README.md

```diff
@@ -23,6 +23,8 @@ None
 * `nagios_server_htpasswd_users.{n}.name`: [required]: Username
 * `nagios_server_htpasswd_users.{n}.password`: [required]: Password
 
+* `nagios_server_htpasswd_realm`: [optional, default `Nagios4`]: Apache2 digest authentication realm
```

Comment:
Maybe mention that this is nagios 4 only?

Reply:
Sure.